caom2/caom2/tests/caom_test_instances.py

```diff
@@ -562,6 +576,14 @@ def get_polarization_wcs(self):
         pol = chunk.PolarizationWCS(axis1d)
         return pol
 
+    def get_custom_wcs(self):
+        axis = wcs.Axis('Foo')
+        axis1d = wcs.CoordAxis1D(axis)
+        axis1d.function = wcs.CoordFunction1D(int(5), 1.0,
+                                              wcs.RefCoord(1.0, 1.0))
+        custom = chunk.CustomWCS(axis1d)
+        return custom
+
```

Comment:
Do there need to be test instances for the ReadGroup fields?

Reply:
Added. Bet it doesn't hurt for code coverage...